stacker/commands/base.py

```diff
@@ -66,3 +67,17 @@ def run(self, options, **kwargs):
 
     def configure(self, options, **kwargs):
         pass
+
+    def get_context_kwargs(self, options, **kwargs):
```

Comment:
well it literally is getting kwargs to pass to the context. adding "additional" implies there are some other kwargs they're being merged with